### ISSUE TITLE
Let users join a submission team also after the deadline

### ISSUE THREAD

**Issue by Splines** [MaMpf-HD/mampf#645]
A user of MaMpf would like to see the following feature (reported via our new Feedback button 🙌):

They complain that it's not possible to join a submission (via a submission code or the link) after the deadline, even if the submission itself was submitted on time by a different teammate. The person would like to join a submission even after the deadline, e.g. in a time frame of 24h. This would still allow them to view the corrected submission directly in MaMpf.

TL;DR: joining a submission (that was handed in on time) should be allowed in a certain timeframe even after the deadline.
